### PR TITLE
added homework_7

### PR DESCRIPTION
**Pull request by mewlunar**

### CODE REVIEW

File: lesson_7_hw/task_2.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+//   Массив должен быть отсортирован по возврастанию количества гласных букв в слове.
+
+function sortString(array) {
+	let tempArray = [...array]
+	tempArray.sort(function (a, b) {
+		return vowelCount(a) - vowelCount(b)
+	})
+
+	return tempArray
+}
```
Comment (VladimirLQA) [inline lesson_7_hw/task_2.js]:
```suggestion
function sortString(array) {
	return [...array].sort(function (a, b) {
		return vowelCount(a) - vowelCount(b)
	})
}
```

File: lesson_7_hw/task_2.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+function vowelCount(string) {
+	const vowels = 'aeijouy'
+	let count = 0
+	for (symbol of string) {
```
Comment (VladimirLQA) [inline lesson_7_hw/task_2.js]:
```suggestion
	for (const symbol of string) {
```

File: lesson_7_hw/task_3.1.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// 5*. Напишите функцию findMissingNumber(arr), которая принимает массив чисел от 1 до N (исключая одно число)
+//   и возвращает пропущенное число. Массив не отсортирован и может содержать дубликаты.
+//   Решите эту задачу, используя эффективные методы массива.
+function findMissingNumber(arr) {
```
Comment (VladimirLQA) [inline lesson_7_hw/task_3.1.js:4]: If interesting, try to refactor for handling the case where two or more numbers are absent

**Review by VladimirLQA — APPROVED** [review]

**Review by anatoly-karpovich — APPROVED** [review]
Well done!

**Pull request merged**